src/components/matrixSize.jsx

```diff
@@ -17,9 +17,9 @@ export default class MatrixSize extends React.Component {
       this.setState({ numCols: event.target.value });
   }
   handleClick(){
-  firebase.database().ref('grids/' + this.state.gridId + '/numRows' ).set(this.state.numRows);
-  firebase.database().ref('grids/' + this.state.gridId + '/numCols' ).set(this.state.numCols);
-
+  firebase.database().ref('grids/' + this.state.gridId + '/numRows' ).set(parseInt(this.state.numRows));
+  firebase.database().ref('grids/' + this.state.gridId + '/numCols' ).set(parseInt(this.state.numCols));
+  console.log()
```

Comment:
I would fix that real quick since it's a related issue. It's really stupid state objects can only be strings